geth/jail/cell_test.go

```diff
@@ -6,40 +6,30 @@ import (
 	"time"
 
 	"github.com/robertkrimen/otto"
+	"github.com/status-im/status-go/geth/jail"
+	"github.com/status-im/status-go/static"
+	"github.com/stretchr/testify/suite"
```

Comment:
You forgot to init testify/suite
`func TestCellTestSuite(t *testing.T) {
	suite.Run(t, new(CellTestSuite))
}`

Let me add it as part of PR #382

Reply:
Thanks!